integration

```diff
@@ -426,6 +426,7 @@
   "duhow/hass-cover-time-based",
   "dummylabs/thewatchman",
   "DurgNomis-drol/ha_toyota",
+  "duwi2024/ha_duwi_home",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>

Reply:
Its not there in the current release.